src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -685,4 +694,18 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    private String generateCoOwnersEmailsLine(String courseId) {
+        List<InstructorAttributes> instructors = instructorsLogic.getInstructorsForCourse(courseId);
+        StringBuffer coOwnersEmailsList = new StringBuffer();
+        for (InstructorAttributes instructor : instructors) {
+            if (!instructor.hasCoownerPrivileges()) {
+                continue;
+            }
+            coOwnersEmailsList.append(instructor.getName() + "(" + instructor.getEmail() + "), ");
+        }
+        return coOwnersEmailsList.length() == 0
+                ? "(No contactable instructors found)"
+                : coOwnersEmailsList.substring(0, coOwnersEmailsList.length() - 2);
```

Comment:
I believe there is a method in StringHelper to help join the strings with a comma, so maybe that method can be used instead.